src/test/java/bio/terra/service/tabulardata/google/BigQueryPdaoUnitTest.java

```diff
@@ -111,25 +113,22 @@
     private BigQueryProject bigQueryProjectDataset;
     @Mock
     private BigQuery bigQueryDataset;
-    @Mock
-    private BigQueryProjectProvider bigQueryProjectProvider;
 
     private Snapshot snapshot;
     private BigQueryPdao dao;
 
     @Before
     public void setUp() throws Exception {
-        MockitoAnnotations.initMocks(this);
```

Comment:
conversely, could have just changed this to `MockitoAnnotations.openMocks(this)`

Reply:
According to the docs `closeMocks()` is needed so I think using the runner is a better approach. https://www.javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/MockitoAnnotations.html

This has the limit of not being able to use other runners though. Another way to do it would be to use [MockitoRule](https://www.javadoc.io/doc/org.mockito/mockito-core/2.2.9/org/mockito/junit/MockitoRule.html). Maybe that's the best approach as it works in all cases?